modules/mono/editor/GodotTools/GodotTools/Build/BuildOutputView.cs

```diff
@@ -1,425 +1,150 @@
+                Flat = true,
+                FocusMode = FocusModeEnum.None,
+                Shortcut = EditorDefShortcut("editor/copy_output", "Copy Selection".TTR(), (Key)KeyModifierMask.MaskCmdOrCtrl | Key.C),
+                ShortcutContext = this,
+            };
+            _copyButton.Pressed += CopyRequested;
+            hbTools.AddChild(_copyButton);
 
-            BuildManager.StopBuild(this);
+            UpdateTheme();
         }
 
-        private enum IssuesContextMenuOption
+        public override void _Notification(int what)
         {
-            Copy
-        }
+            base._Notification(what);
 
-        private void IssuesListContextOptionPressed(long id)
-        {
-            switch ((IssuesContextMenuOption)id)
+            if (what == NotificationThemeChanged)
             {
-                case IssuesContextMenuOption.Copy:
-                {
-                    // We don't allow multi-selection but just in case that changes later...
-                    string text = null;
-
-                    foreach (int issueIndex in _issuesList.GetSelectedItems())
-                    {
-                        if (text != null)
-                            text += "\n";
-                        text += _issuesList.GetItemText(issueIndex);
-                    }
-
-                    if (text != null)
-                        DisplayServer.ClipboardSet(text);
-                    break;
-                }
-                default:
-                    throw new ArgumentOutOfRangeException(nameof(id), id, "Invalid issue context menu option");
+                UpdateTheme();
             }
         }
 
-        private void IssuesListClicked(long index, Vector2 atPosition, long mouseButtonIndex)
+        private void UpdateTheme()
         {
-            if (mouseButtonIndex != (long)MouseButton.Right)
-            {
+            // Nodes will be null until _Ready is called.
+            if (_log == null)
                 return;
-            }
-
-            _ = index; // Unused
-
-            _issuesListContextMenu.Clear();
-            _issuesListContextMenu.Size = new Vector2I(1, 1);
-
-            if (_issuesList.IsAnythingSelected())
-            {
-                // Add menu entries for the selected item
-                _issuesListContextMenu.AddIconItem(GetThemeIcon("ActionCopy", "EditorIcons"),
-                    label: "Copy Error".TTR(), (int)IssuesContextMenuOption.Copy);
-            }
-
-            if (_issuesListContextMenu.ItemCount > 0)
-            {
-                _issuesListContextMenu.Position = (Vector2I)(_issuesList.GlobalPosition + atPosition);
-                _issuesListContextMenu.Popup();
-            }
-        }
-
-        public override void _Ready()
-        {
-            base._Ready();
 
-            SizeFlagsVertical = SizeFlags.ExpandFill;
+            var normalFont = GetThemeFont("output_source", "EditorFonts");
```

Comment:
Here and below, did you copy verbatim the same override setup from `editor_log`?

Reply:
I did yes, my goal was to have a consistent look.

Reply:
Okay, I think it needs to be converted to a theme type variation. I'll keep in mind that you're relying on the same styles as well when I do that.